.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -0,0 +1,70 @@
+name: Issue Report
+description: Report a Issue
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello, Please fill out the sections below to help everyone identify and fix the bug
+  - type: checkboxes
+    id: Vaildation
+    attributes:
+      label: Validation Checklist
+      options:
+        - label: Is this a Kubeflow issue?
+          required: true
+        - label: Is this for Kubeflow development ?
+          required: true
+        - label: Are you posting in the right repository ?
+          required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 1.8
+        - 1.7
+        - 1.6
+        - 1.5
+        - 1.4
+        - 1.3
+        - 1.2
+        - 1.1
+        - 1.0
+        - 0.7
+        - 0.6
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe your issue
+      placeholder: When I Procced this issue is occured. 
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce the issue
+      placeholder: |
+        1. Try this ...
+        2. Then do this ...
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Put here any screenshots or videos (optional)
+  - type: dropdown
+    id: assign
+    attributes:
+      label: "Would you like to work on this issue?"
+      options:
+        - "Yes"
+  - type: markdown
+    attributes:
+      value: |
+        > **Note:** Please note that you have to fill required fields to post an Issue.
```

Comment:
Hello,

1. Is this a Kubeflow issue ?
2. Is this for Kubeflow development ?
3. Are you posting in the right repository ?
4. Did you follow the installation guide https://github.com/kubeflow/manifests?tab=readme-ov-file ? 
5. Is the issue report properly structured and detailed with version numbers ?

If not please close or redo the issue and join our slack channel `wg-manifests` here https://www.kubeflow.org/docs/about/community/. This link also contains our meeting schedule.

This should be our checklist.